charts/argo-cd/crds/crd-extension.yaml

```diff
@@ -1,96 +0,0 @@
-apiVersion: apiextensions.k8s.io/v1
```

Comment:
Why did you remove this CRD?

Reply:
Seems this was an oversight copying new ones from upstream :thinking:

Reply:
Ok 👌 , I filed a new PR to fix this 🚀 https://github.com/argoproj/argo-helm/pull/1067